crates/edr_eth/src/transaction.rs

```diff
@@ -234,6 +234,11 @@ pub trait IsEip4844 {
     fn is_eip4844(&self) -> bool;
 }
 
+pub trait IsExecutable {
+    /// Whether the transaction is executable.
+    fn is_executable_transaction(&self) -> bool;
```

Comment:
nit: The call sites seem to assume that "executable" specifically means unsupported. Maybe we should rename it to `is(_transaction)_supported` or similar? I had to look at the call sites to understand why a transaction might not be executable, so maybe referring to the supported/recognized quality directly here would make it more obvious.

Reply:
Addressed in https://github.com/NomicFoundation/edr/pull/671/commits/9d3adad2a37095385ca9a6f7ff5c9c719e842f2c